libevmasm/ZKEVMInstruction.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+
+#include <libevmasm/ZKEVMInstruction.h>
+
+using namespace solidity::zkevm;
+
+const std::array<MinInstructionInfo, 25> solidity::zkevm::instructionInfos{
+	{{"to_l1", 3, 0},
```

Comment:
It's just come to my mind that we need some prefixes for our instructions to easily distinguish them in the new code and not to create collisions with existing user-defined functions.

Please add $zk_ prefix to each instruction. The dollar sign must be ok IIRC.